src/qmapshack/poi/CPoiPOI.cpp

```diff
@@ -86,126 +88,269 @@ void CPoiPOI::draw(IDrawContext::buffer_t& buf)
         xMax =  180 * DEG_TO_RAD;
     }
 
+#ifndef QT_NO_DEBUG_STREAM
```

Comment:
@kiozen is this the way this should be done for debug purposes? Or should I completely remove the timing again?

Reply:
hm, so far I commented out debug logging if I thought it might be of later use.  But timing stuff is seldom used twice. I would remove it. It's easy to add these 3 lines again on-the-fly.